src/app/components/Share/Button.tsx

```diff
@@ -1,15 +1,19 @@
 import { IconArrowRight } from "@/app/icons/IconArrowRight";
 
-const Button = () => {
+interface ButtonProps {
+  href: string;
+}
+
+const Button: React.FC<ButtonProps> = ({ href }) => {
   return (
     <a
-      href="/"
-      className="group flex w-max items-center gap-[10px] rounded-full bg-gradient-btn px-[24px] py-[12px] shadow-button-shadow transition duration-300 ease-in-out hover:scale-[1.05] focus:scale-[1.05] group-hover:text-violet group-focus:text-violet lg:px-[32px] lg:py-[16px]"
+      href={href}
+      className="shadow-button-shadow bg-gradient-btn group-hover:text-violet group-focus:text-violet group flex max-w-[219px] items-center gap-[10px] rounded-full px-[24px] py-[12px] transition duration-300 ease-in-out hover:scale-[1.05] focus:scale-[1.05]"
     >
-      <span className="font-medium leading-[1.5] transition duration-300 ease-in-out group-hover:text-violet group-focus:text-violet lg:text-lg lg:font-semibold">
+      <span className="group-hover:text-violet group-focus:text-violet font-medium leading-[1.5] transition duration-300 ease-in-out">
```

Comment:
tu jeszcze trzeba coś pomyśleć nad tym, żeby text buttona też dostosowywał się do widoków mobile i desktop

Reply:
Właśnie nie wiem, czy już nie zostawić dla mobile takiego samego, jak dla desktopa - tak na razie zrobiłam i dobrze to wygląda, nie widać mocno różnicy